zio-http/shared/src/main/scala/zio/http/Body.scala

```diff
@@ -531,8 +531,8 @@ object Body {
 
   }
 
-  private val zioEmptyArray = ZIO.succeed(Array.empty[Byte])(Trace.empty)
+  private val zioEmptyArray = Exit.succeed(Array.emptyByteArray)
```

Comment:
Note on this: Empty arrays are effectively immutable because they have a fixed length of 0, and so they cannot be mutated